Editor/Menus/CinemachineMenu.cs

```diff
@@ -83,7 +83,8 @@ private static void CreateBlendListCamera()
             vcam.m_Instructions[1].m_Blend.m_Style = CinemachineBlendDefinition.Style.EaseInOut;
             vcam.m_Instructions[1].m_Blend.m_Time = 2f;
         }
-
+        
+#if CINEMACHINE_UNITY_ANIMATION
```

Comment:
Need at the top:
#if !UNITY_2019_3_OR_NEWER
#define CINEMACHINE_UNITY_ANIMATION
#endif